src/amo/components/AddonMoreInfo.js

```diff
@@ -16,21 +17,33 @@ export class AddonMoreInfoBase extends React.Component {
   render() {
     const { addon, i18n } = this.props;
 
-    let homepage = addon.homepage ? addon.homepage.trim() : null;
-    if (homepage && !homepage.match(/^https?:\/\//)) {
-      homepage = `http://${homepage}`;
-    }
+    const homepage = trimAndAddProtocolToUrl(addon.homepage);
+    const supportUrl = trimAndAddProtocolToUrl(addon.support_url);
 
     return (
       <Card className="AddonMoreInfo" header={i18n.gettext('More information')}>
         <dl className="AddonMoreInfo-contents">
-          {addon.homepage ? <dt>{i18n.gettext('Website')}</dt> : null}
-          {homepage ? (
-            <dd>
-              <a href={homepage}
-                ref={(ref) => { this.homepageLink = ref; }}>
-                {homepage.replace(/^https?:\/\//, '')}
-              </a>
+          {homepage || supportUrl ? (
```

Comment:
It might be cleaner to move these bits into separate snippets using if statements to avoid so much ternary in the JSX?